app/src/main/java/org/coepi/android/cen/CENApi.kt

```diff
@@ -9,8 +9,8 @@ import retrofit2.http.Path
 
 interface CENApi {
     // post CENReport along with CENKeys
-    @POST("/cenreport")
-    fun postCENReport(@Body report : RealmCenReport): Single<Unit>
+    @POST("/cenreport/13298327ebcebe7f153b956e4596d503")
```

Comment:
Use path variable https://square.github.io/retrofit/

Reply:
the part after /132.... it's not needed I've removed